src/RunProcessAsTask.Tests/RunProcessAsTask.Tests.csproj

```diff
@@ -1,16 +1,20 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">

```

Comment:
It's possibly confused and trying to use old tooling?  I'd remove it; you don't want that to be possible.  I've never seen a new-style project file with an msbuild; they all start with `<Project Sdk="Microsoft.NET.Sdk">`.